docs/api/fr/materials/Material.html

```diff
@@ -350,7 +350,9 @@ <h3>[method:undefined dispose]()</h3>
 		<h3>[method:undefined onBeforeCompile]( [param:Shader shader], [param:WebGLRenderer renderer] )</h3>
 		<p>
 			Un callback facultatif qui est exécuté immédiatement avant la compilation du programme shader.
-			Cette fonction est appelée avec le code source du shader comme paramètre. Utile pour la modification des matériaux intégrés.
+			Cette fonction est appelée avec le code source du shader comme paramètre. Utile pour la modification des matériaux intégrés, 
+			bien que la nouvelle méthode recommandée soit d'utiliser `WebGPURenderer` avec le nouveau système de Node Material et
```

Comment:
> avec le nouveau système de Node Material

This feels a bit weird, but translating Node Material in french might be even weirder. Wdyt?
Maybe a link to this new Node Material would make things more natural? 

Is it part of the doc that is WIP? Or can I link to it now?

Reply:
You can't link to it yet since it is too early to host the new docs. For now, I suggest you don't translate the term an use Node Material.

Reply:
Got it, so all good then?